packages/prettier-config/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@open-wc/prettier-config",
+  "version": "0.3.1",
+  "description": "Prettier config following open-wc recommendations",
+  "author": "open-wc",
+  "homepage": "https://github.com/open-wc/open-wc/",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": "https://github.com/open-wc/open-wc/tree/master/packages/prettier-config",
+  "main": ".prettierrc.js",
+  "dependencies": {
+    "eslint-config-prettier": "^3.3.0",
+    "husky": "^1.2.0",
```

Comment:
I would say husky & lint-staged should not be dependencies of the config itself.
We should, however, explain how to set it up in the readme.

=> not (needed) for this MR (unless you are really eager to work on it) but in the long run we should have a "meta" package `linting` (like we have for testing) which will have a dependency on husky/lint-staged and that will be the entry point for most users.

Reply:
Let's consider adding these dependencies in scope of separate PR.

Reply:
sounds good :)